src/Bridge/Symfony/Routing/IriConverter.php

```diff
@@ -92,7 +92,7 @@ public function getItemFromIri(string $iri, array $context = [])
     public function getIriFromItem($item, int $referenceType = UrlGeneratorInterface::ABS_PATH): string
     {
         $resourceClass = $this->getObjectClass($item);
-        $routeName = $this->routeNameResolver->getRouteName($resourceClass, false);
+        $routeName = $this->routeNameResolver->getRouteName($resourceClass, OperationType::ITEM, []);
```

Comment:
Thanks @soyuka I have removed the trigger error from both the CachedRouteNameResolver and  RouteNameResolver. 

I'm not familiar with the syntax of the optional argument in this interface/class - if there is no deprecation to handle can the method signatures also be refactored?

```php
public function getRouteName(string $resourceClass, $operationType /**, array $context = [] **/): string
```
changed to 
```php
public function getRouteName(string $resourceClass, $operationType , array $context = [] ): string
```
and drop this bit?
```php
        if (func_num_args() > 2) {
            $context = func_get_arg(2);
        } else {
            $context = [];
        }
```

In CachedRouteNameResolver, RouteNameResolver and RouteNameResolverInterface?